iron-doc-property.html

```diff
@@ -93,6 +93,15 @@
         observer: '_collapsedChanged',
       },
 
+      /**
+       * Unique anchor ID for deep-linking.
+       *
+       */
+      id: {
```

Comment:
Yeah, it used to be called "anchorId" -- I was trying to get around having two IDs here (we originally had "id" on the anchor tag inside the iron-doc-property, my solution needs an attribute on the iron-doc-property itself so that iron-doc-viewer can see it with querySelector without using /deep/).

But since the catalog _is_ using /deep/, I could just change this back to anchorId and put an ID on the anchor tag, instead.